bigchaindb/backend/exceptions.py

```diff
@@ -0,0 +1,5 @@
+from bigchaindb.exceptions import BigchainDBError
+
+
+class DatabaseOpFailedError(BigchainDBError):
```

Comment:
Oh, ok. I thought we wanted at some point to re-organize the exceptions such that those which are server specific are not in common, among other things.